cgroup-limit/test.json

```diff
@@ -7,7 +7,5 @@
   "type": "bash",
   "cleanup": true,
   "ignoredRIDs":[
-    "linux-s390x",
-    "linux-ppc64le"
```

Comment:
I am not sure if this is the best way to go about this. This would enable the tests s390x and ppc64le even on older .NET releases, like 6.0. We should still skip the test for 6.0, somehow.